src/components/Form.vue

```diff
@@ -0,0 +1,36 @@
+<template>
+  <div>
+    <b-field label="Enter Todo..">
+      <b-input v-model="message" @keypress.native.enter="addTodo"></b-input>
+    </b-field>
+  </div>
+</template>
+<script>
+export default {
+  name: "Form",
+  computed: {
+    message: {
+      set: function(val) {
+        this.$store.commit("updateMessage", val);
+      },
+      get: function() {
+        return this.$store.state.todo.addTodo.message;
```

Comment:
use vuex helper classes & shorted this "this.$store.state.todo.addTodo.message;"